### PR TITLE
Use assertj fluent style in flowable-idm-engine.

### PR DESCRIPTION
**Pull request by dbmalkovsky**
All test files in the `org.flowable.idm.engine.test` package are included.

Converted a mixture of Junit 4 and 5 and `assertj` code into all Junit 5 and  `assertj` statements.

### CODE REVIEW

File: modules/flowable-idm-engine/src/test/java/org/flowable/idm/engine/test/api/identity/GroupQueryTest.java [hunk 1]
```diff
@@ -179,150 +165,130 @@ public void testQueryByMember() {
...
         query = query.orderByGroupId().asc();
         List<Group> groups = query.list();
-        assertEquals(3, groups.size());
-        assertEquals("admin", groups.get(0).getId());
-        assertEquals("frogs", groups.get(1).getId());
-        assertEquals("muppets", groups.get(2).getId());
+        assertThat(groups).hasSize(3);
+        assertThat(groups.get(0).getId()).isEqualTo("admin");
+        assertThat(groups.get(1).getId()).isEqualTo("frogs");
+        assertThat(groups.get(2).getId()).isEqualTo("muppets");
```
Comment (filiphr) [inline modules/flowable-idm-engine/src/test/java/org/flowable/idm/engine/test/api/identity/GroupQueryTest.java:171]: Can we replace constructs like this (testing order of elements) with something like:

```
assertThat(groups).containsExactly("admin", "frogs", "muppets");
```

File: modules/flowable-idm-engine/src/test/java/org/flowable/idm/engine/test/api/identity/TokenQueryTest.java [hunk 2]
```diff
@@ -301,82 +274,71 @@ public void testQueryByInvalidTokenDataLike() {
...
 
         // Combined with criteria
         TokenQuery query = idmIdentityService.createTokenQuery().userAgentLike("%firefox%").orderByTokenDate().asc();
         List<Token> tokens = query.list();
-        assertEquals(2, tokens.size());
-        assertEquals("firefox", tokens.get(0).getUserAgent());
-        assertEquals("firefox2", tokens.get(1).getUserAgent());
+        assertThat(tokens).hasSize(2);
+        assertThat(tokens.get(0).getUserAgent()).isEqualTo("firefox");
+        assertThat(tokens.get(1).getUserAgent()).isEqualTo("firefox2");
```
Comment (filiphr) [inline modules/flowable-idm-engine/src/test/java/org/flowable/idm/engine/test/api/identity/TokenQueryTest.java:296]: Here we can use a construct like:

```
assertThat(tokens)
    .extracting(Token::getUserAgent)
    .containsExactly("firefox", "firefox2");
```

**Comment by dbmalkovsky** [comment]
I've updated all the marked sections and found similar code in other files too.  Should be good to go.

**Comment by filiphr** [comment]
Thanks David

**Pull request merged**